### ISSUE TITLE
release v0.9.0

### ISSUE THREAD

**Issue by mathause** [regionmask/regionmask#302]
There are some notable enhancements (e.g. #280) and some fixes (e.g. #289, #291) - so would be good to get a release out. Also my next project - #228 - is a (slightly) larger undertaking - let's not delay the release much longer because of this.

Should be addressed:
- #301
- #265
- #300
- #306

Nice to have:
- #298
- #244 (depends on #265)
